content/en/blog/2025/otel-mainframe-priorities-survey.md

```diff
@@ -0,0 +1,64 @@
+---
+title: OpenTelemetry on Mainframe Priorities Survey
+linkTitle: Mainframe Priorities Survey
+date: 2025-01-24
```

Comment:
```suggestion
date: 2025-01-30
```

let's keep this open until we set the right date.

Reply:
@rrschulze let's keep this open until we decide on a final schedule, might push out to tomorrow, so I want to ensure that we have the right date set.